app/views/govuk_publishing_components/components/_back_link.html.erb

```diff
@@ -1,4 +1,7 @@
-<%= link_to t('components.back_link.back'),
+<%
+  text ||= t('components.back_link.back')
+%>
+<%= link_to text,
```

Comment:
This looks good - but it'd be worth having this as an example for the component page on https://govuk-publishing-compon-pr-628.herokuapp.com/component-guide/back_link

Here's an example of a component with different examples: https://govuk-publishing-compon-pr-628.herokuapp.com/component-guide/breadcrumbs

Reply:
I've added an example 👍